### PR TITLE
Build confidence settings

### PR DESCRIPTION
**Pull request by jscook2345**
models: add the build confidence project attributes

### CODE REVIEW

File: squad_client/commands/create_or_update_project.py [hunk 1]
```diff
@@ -62,6 +62,16 @@ def register(self, subparser):
             "--important-metadata-keys",
             help="Important metadata keys separated by comma",
         )
+        parser.add_argument(
+            "--build-confidence-count",
+            help="",
```
Comment (chaws) [inline squad_client/commands/create_or_update_project.py:67]: Could you add the help text for this one?

File: squad_client/commands/create_or_update_project.py [hunk 2]
```diff
@@ -62,6 +62,16 @@ def register(self, subparser):
...
+            type=int,
+        )
+        parser.add_argument(
+            "--build-confidence-threshold",
+            help="",
```
Comment (chaws) [inline squad_client/commands/create_or_update_project.py:72]: Same as above

**Comment by chaws** [comment]
LGTM, just add the help text as I commented above then I think it's good to go.

File: squad_client/commands/create_or_update_project.py (hunk 1 above)
Comment (jscook2345) [reply squad_client/commands/create_or_update_project.py]: Done!

File: squad_client/commands/create_or_update_project.py (hunk 2 above)
Comment (jscook2345) [reply squad_client/commands/create_or_update_project.py]: Done!

**Pull request merged**